release_notes.md

```diff
@@ -1,6 +1,30 @@
-# Release Notes
+# Release Note 0.2.0
 
-## Feature Highlight
+## ONE-vscode
 
-## Improvements
+### OneExplorerView
 
+- Support more commands
+  - [Open with Text Editor] command
+  - [Open Containing Folder] command
+  - [Collapse All] command
+- [Improve Create Config] command
+- Prevent errors on soft link / virtualenv .cfg file
+- Rename related files automatically on cfg edit
```

Comment:
These are not implemented yet 😢

Reply:
Oh, I just copy from https://github.com/Samsung/ONE-vscode/issues/712#issuecomment-1138098574 . I will fix it.